tools/depends/target/rapidjson/Makefile

```diff
@@ -1,6 +1,8 @@
 -include ../../Makefile.include
 include RAPIDJSON-VERSION
-DEPS = Makefile RAPIDJSON-VERSION 0001-remove_custom_cxx_flags.patch ../../download-files.include
+DEPS = Makefile RAPIDJSON-VERSION 0001-remove_custom_cxx_flags.patch ../../download-files.include \
+                                  002-cmake-removedocs-examples.patch \
+                                  003-win-arm64.patch
```

Comment:
New patch file names are missing a leading zero. No biggie, just to maintain consistency.